### PR TITLE
Feat swift generics

### PR DESCRIPTION
**Pull request by abnegate**
## What does this PR do?

Add Swift generics support

## Test Plan

Existing tests

## Related PRs and Issues

https://github.com/appwrite/sdk-generator/pull/559

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/SDK/Language.php [hunk 1]
```diff
@@ -83,4 +83,18 @@ public function getFilters(): array
     {
         return [];
     }
+
+    protected function toUpperCase(string $value): string
```
Comment (lohanidamodar) [inline src/SDK/Language.php:87]: I think the better name would be `toUpperCamelCase` instead? WDYT?

**Review by lohanidamodar — COMMENTED** [review]
Looks much better. would love to see the Generated SDK and left a suggestion as well.

File: src/SDK/Language.php (hunk 1 above)
Comment (abnegate) [reply src/SDK/Language.php]: I have renamed to `toUpperCaseWords` and `toCamelCase`, I think that's more clear

**Comment by abnegate** [comment]
@lohanidamodar Generated SDK can be compared here: https://github.com/appwrite/sdk-for-apple/compare/master...dev

**Review by lohanidamodar — APPROVED** [review]
Looks good.

**Review by lohanidamodar — APPROVED** [review]

**Pull request merged**